tsdb/ooo_head_read.go

```diff
@@ -111,7 +107,7 @@ func getOOOSeriesChunks(s *memSeries, mint, maxt int64, lastGarbageCollectedMmap
 					return nil
 				}
 				for _, chk := range chks {
-					addChunk(c.minTime, c.maxTime, ref, chk.chunk)
+					addChunk(chk.minTime, chk.maxTime, ref, chk.chunk)
```

Comment:
similar; this was a bug?

Reply:
In theory yes, but in practice float samples don't result in multiple chunks.